tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py

```diff
@@ -0,0 +1,150 @@
+    will launch an initial scan (BASELINE_SCAN) on it. Then the system date is changed to a time after
+    the date of the initial baseline scan, but before the minimum period set in "min_full_scan_interval",
+    so that the full scan should not be triggered. Finally, the date is changed again to a time when
+    the period set in "min_full_scan_interval" has expired, at which point the full scan should be triggered.
+
+    Args:
+        configure_local_internal_options (fixture): Set internal configuration for testing.
+        get_configuration (fixture): Get configurations from the module.
+        configure_environment (fixture): Configure a custom environment for testing.
+        restart_modulesd (fixture): Reset ossec.log and start a new monitor.
+        add_simulated_agent (fixture): Add a simulated agent to the manager for testing.
+    """
+    check_apply_test({'min_full_scan_interval'}, get_configuration['tags'])
+    config = get_configuration['metadata']
+    agent_id = add_simulated_agent
+
+    # Callbacks
+    callback_detect_full_scan_start = vd.make_vuln_callback(f"A full scan will be run on agent '{agent_id}'")
+    callback_detect_baseline_scan_start = vd.make_vuln_callback(f"A baseline scan will be run on agent '{agent_id}'")
+    callback_detect_scan_end = vd.make_vuln_callback(f"Finished vulnerability assessment for agent '{agent_id}'")
+
+    min_full_scan_interval = time_to_seconds(config['min_full_scan_interval'])
+    seconds_to_travel = min_full_scan_interval / 2
+
+    # Detect baseline scan.
+    wazuh_log_monitor.start(timeout=vd.VULN_DETECTOR_BASELINE_SCAN_TIMEOUT,
+                            callback=callback_detect_baseline_scan_start,
+                            error_message='No baseline scan detected in log.')
+
+    # Detect baseline scan completion.
+    wazuh_log_monitor.start(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
+                            callback=callback_detect_scan_end,
+                            error_message='No scan end detected in log.')
+
+    # Travelling in time, halfway through the period set in "min_full_scan_interval".
+    before = str(datetime.now())
+    TimeMachine.travel_to_future(timedelta(seconds=seconds_to_travel))
+    logger.debug(f"Changing the system clock from {before} to {datetime.now()}")
+
+    # Since the time set in "min_full_scan_interval" has not yet expired, the full scan should not be triggered.
+    with pytest.raises(TimeoutError):
+        wazuh_log_monitor.start(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
+                                callback=callback_detect_full_scan_start,
+                                error_message='No full scan start detected in log.')
+
+    # Travelling in time, at the moment when the period set in "min_full_scan_interval" has expired.
+    # From this moment the full scan should be triggered.
+    before = str(datetime.now())
```

Comment:
Same here.

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1369/commits/5045a8250c712c9081ddc6565b62c4c87b998a5c